README.md

```diff
@@ -1,2 +1,34 @@
 # school-library
 In this project, you will start building your school library app. In this initial step, you will implement the classes to represent students and teachers.
+
+## Getting Started
+
+This repository includes files with plain ruby
+
+
+## Authors
+
+👤 **Karam Fouad**
+- GitHub: [@githubhandle](https://github.com/karam084)
+- Twitter: [@twitterhandle](https://twitter.com/ElarabFouad)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/karam-fouad-179830214/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jaferIdrees/vet_clinic_db/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] the MIT link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉
> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

Reply:
Thank you, I will do